src/plugins/data/public/antlr/dql/code_completion.ts

```diff
@@ -247,14 +247,21 @@ export const getSuggestions = async ({
                 cursorLine,
                 cursorColumn + 1
               ),
-              insertText: `${val} `,
+              insertText: `"${val}" `,
             };
           })
         );
       }
     }
 
-    const dqlOperators = new Set([DQLParser.AND, DQLParser.OR, DQLParser.NOT]);
+    const booleanOperators = new Set([DQLParser.AND, DQLParser.OR, DQLParser.NOT]);
+    const operators = new Set([
```

Comment:
nit. may use more descriptive naming here, something like `singleQuoteWrappedOperators` to align with the above `booleanOperators` naming

Reply:
changed it to relationalOperators